en/user_guide/workflow.rst

```diff
@@ -43,39 +45,51 @@ The status form asks for the following information:
 * **Name** (required): status name
 * **Published**: if this status is final
 * **Initial**: if this status is initial
-* **Label**: status label in all available languages
+* **Label**: status label in all available languages in Back Office
 * **Color** (required): color representing the status (used on the node or content edit page)
 
 .. image:: ../../images/status_form.png
 
-Transitions
------------
-
-A transitions links two status together, for instance in the previous example, there is a transition
-to change an article from Draft to Pending and another one to change it from Pending to Published.
-
-As not all contributors may be allowed to change the status of an article, there is a right notion
-related to the transitions. In fact in Open Orchestra, transitions are rights. So to create a new
-transition an administrator only have to create a new right. In Open Orchestra, rights are called
-Roles.
+Roles
+-----
 
 The role form asks for the following information:
 
 * **Name**: role name
-* **Description**: role description in all available languages.
+* **Description**: role description in all available languages in Back Office.
+* **Status from**: as roles are transitions between two statuses, source status for the role
+* **Status destination**: as roles are transitions between two statuses, destination status for the role
 
 .. image:: ../../images/role_form.png
 
-The selection of two status in the role form will define a transition role, and implicitly this transition.
+The selection of two statuses in the role form will define a workflow role.
 
 .. image:: ../../images/role_transition.png
 
 This role creates a transition between Draft status and Pending status. So users with this role will
 be able to change the status of a node or content from Draft to Pending. They will do this via a small
-widget on the node edit page.
+widget on the node or content edit page.
 
 .. image:: ../../images/status_transition.png
 
-See also `group management`_ to know how to give roles to the users.
+If no statuses are selected, the role will define an access role. See also `group management`_
+
+Workflow Function
+-----------------
+
+The workflow function form asks for the following information:
+
+* **Name**: workflow function name in all available languages in Back Office.
+* **Role**: a list of all roles with a from and destination status, multiple selection allowed.
+
+.. image:: ../../images/workflow_function_form.png
```

Comment:
Roles have been recently translated, so I suggest to update that screen to make constants disappear